src/service/chat.js

```diff
@@ -206,19 +206,19 @@ const moveChatToAnon = async ({
   }
 };
 
-const initChatFromPost = async ({source, id}) => {
+const initChatFromPost = async ({source, id, postId}) => {
   let payload = {source};
   switch (source) {
     case 'post':
       payload = {...payload, postId: id};
       break;
     case 'comment':
-      payload = {...payload, commentId: id};
+      payload = {...payload, commentId: id, postId};
+      console.log('payload', payload);
       break;
     default:
       break;
   }
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `initChatFromPost` function has been updated to include a new parameter `postId`. This is used in the case where the source is 'comment'. However, there's a console log statement on line 217 which should be removed as it's not good practice to leave debugging logs in production code.
```diff
-       console.log('payload', payload);
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->